.github/workflows/act/pull-request-event.json

```diff
@@ -0,0 +1,5 @@
+{
+  "label": {
+    "name": "staging"
+  }
+}
```

Comment:
I'm not sure I understand the purpose of this file

Reply:
I was using it for running Actions locally via [act](https://github.com/nektos/act), but since it's a pain to use _(because Act does not yet support reusable workflows)_ then perhaps I should just remove this file instead of attempting to document an inelegant process.